Dockerfile

```diff
@@ -15,7 +16,7 @@ RUN DEBIAN_FRONTEND=noninteractive \
  && SUFFIX='' \
  && if [ "$MINIMAL" = "true" ]; then SUFFIX="${SUFFIX}-minimal"; fi \
  && if [ "$IOG" = "true" ]; then SUFFIX="${SUFFIX}-iog"; fi \
- && if [ "$IOG_FULL" = "true" ]; then SUFFIX="${SUFFIX}-iog-full"; fi \
+ && if [ "$IOG_FULL" = "true" ]; then SUFFIX="${SUFFIX}-full"; fi \
```

Comment:
Not `-iog-full`? 😕

Reply:
I'll do a follow-up PR that refactor that!